File: python/prob2.py
#!/usr/bin/env python
#By considering the terms in the Fibonacci sequence whose values do not exceed four million find the
#sum of the even-valued terms

first_number = 1
second_number = 1
fib = 0
total = 0

while fib < 4000000:
    fib = first_number + second_number
    if fib % 2 == 0:
        total += fib
    first_number = second_number
    second_number = fib

print(total)

File: go/prob2.go
//By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.

package main

import "fmt"

func main() {
	var (
		num1 = 1
		num2 = 1
		fib = 0
		total = 0
	)
	for fib < 4000000 {
		fib = num1 + num2
		if fib % 2 == 0 {
			total += fib
		}
		num1 = num2
		num2 = fib
	}
	fmt.Println(total)
}
